src/main/java/frc/robot/commands/ChargeStationBalance.java

```diff
@@ -88,7 +95,7 @@ public void end(boolean interrupted) {
   @Override
   public boolean isFinished() {
     //if robot is balanced and it has been for one second, robot ends
-    if (m_balanced = true && m_timer.get() == m_curTime + Constants.CHARGESTATION_BALANCE_SECONDS){
+    if (m_balanced = true && m_timer.get() == m_curTime + m_balanceSeconds){
```

Comment:
"==" should never appear because it is never going to be exact accurate when we read from robot sensors or internal timer (the timer class reads time using the system clock on your computer system i believe). So it should be <=